src/containers/apps/CreateNewApp.tsx

```diff
@@ -89,23 +103,39 @@ class CreateNewApp extends Component<
                                     })
                                 }
                             >
-                                Has Persistent Data
+                                {localize(
+                                    'create_new_app.has_persistent_data',
+                                    'Has Persistent Data'
+                                )}
                             </Checkbox>
                             &nbsp;&nbsp;
-                            <Tooltip title="Mostly used for databases, see docs for details.">
```

Comment:
The original tooltip not working, so I adjusted the positions of NewTabLink and Tooltip.